src/components/rtc-settings-panel.tsx

```diff
@@ -103,36 +94,52 @@ export function RTCSettingsPanel() {
 					<ToggleControl
 						label="Enable avatars"
 						checked={ isAvatarsEnabled }
-						onChange={ ( enabled: boolean ) => {
-							handleToggleAvatars( enabled );
-						} }
+						onChange={ ( enabled: boolean ) => setSetting( Setting.AWARENESS_AVATARS, enabled ) }
 					/>
 
 					<ToggleControl
 						label="Enable cursors"
 						checked={ isCursorsEnabled }
-						onChange={ ( enabled: boolean ) => {
-							handleToggleCursors( enabled );
-						} }
+						onChange={ ( enabled: boolean ) => setSetting( Setting.AWARENESS_CURSORS, enabled ) }
 					/>
 
 					<ToggleControl
 						label="Show my awareness"
 						checked={ isSelfAwarenessEnabled }
-						onChange={ ( enabled: boolean ) => {
-							handleToggleSelfAwareness( enabled );
-						} }
+						onChange={ ( enabled: boolean ) => setSetting( Setting.SELF_AWARENESS, enabled ) }
 					/>
 
 					{ isDevelopment() && (
 						<ToggleControl
 							label="Show debug tools"
 							checked={ isDebugToolsEnabled }
-							onChange={ ( enabled: boolean ) => {
-								handleToggleDebugTools( enabled );
-							} }
+							onChange={ ( enabled: boolean ) => setSetting( Setting.DEBUG_TOOLS, enabled ) }
 						/>
 					) }
+
+					<Heading level={ 3 } style={ { marginTop: '24px' } }>
+						{ __( 'Notifications', 'vip-real-time-collaboration' ) }
+					</Heading>
+
+					<Heading level={ 2 } style={ { marginBottom: '2px' } }>
+						{ __( 'Collaborators', 'vip-real-time-collaboration' ) }
+					</Heading>
+
+					<ToggleControl
+						label="Enters"
+						checked={ isUserEnterNotificationEnabled }
+						onChange={ ( enabled: boolean ) =>
+							setSetting( Setting.USER_ENTER_NOTIFICATION, enabled )
+						}
+					/>
+
+					<ToggleControl
+						label="Exits"
+						checked={ isUserExitNotificationEnabled }
+						onChange={ ( enabled: boolean ) =>
+							setSetting( Setting.USER_EXIT_NOTIFICATION, enabled )
+						}
+					/>
```

Comment:
Do you think these should be a single setting? Why would someone want one and not the other?